docs/source/whatsnew/0.9.1.txt

```diff
@@ -27,6 +27,13 @@ Enhancements
   which manages expiration of entries based on the `dt` supplied to the `get`
   method.
 
+* Implemented :class:`zipline.pipeline.factors.RecarrayFactor`, a new pipeline
+  term designed to be the output type of a CustomFactor with multiple outputs.
```

Comment:
This should reference the issue number.

Also, I think the next release is going to be 1.0, not 0.9.1, so we should probably move the notes here to 1.0.0.txt